schemas/json/block.json

```
@@ -739,7 +739,7 @@
 				},
 				"innerBlocks": {
 					"type": "array",
-					"description": "Set the inner blocks that should be used within the block example. The blocks should be defined as a nested array like this: \n\n [ [ 'core/heading', { content: 'This is an Example' }, [] ] ]\n\n Where each block itself is an array that contains the block name, the block attributes, and the blocks inner blocks."
+					"description": "Set the inner blocks that should be used within the block example. The blocks should be defined as a nested array like this:\\n\\n[{ name: 'core/heading', innerBlocks: [{ name: 'core/subheading', attributes: { content: 'This is an example.' }}, { }] }]\\n\\nWhere each block itself is an object that contains the block name, the block attributes, and the blocks inner blocks."
```

```suggestion
					"description": "Set the inner blocks that should be used within the block example. The blocks should be defined as a nested array like this:\\n\\n[ { \"name\": \"core/heading\", \"innerBlocks\": [ { \"name\": \"core/heading\", \"attributes\": { \"content\": \"This is an Example\" } } ] } ]\\n\\nWhere each block itself is an object that contains the block name, the block attributes, and the blocks inner blocks."
```



This code is expected to be written in block.json. Therefore, single quotes cannot be used, and properties and values must be wrapped in double quotes.

Additionally, there is no "subheading" block in the core. I think It is better to keep with the original example.

---

Sorry, there seems to be an error in the code I suggested 😅 Could you update this line as follows just one more time? This should be correct.

```
"description": "Set the inner blocks that should be used within the block example. The blocks should be defined as a nested array like this:\n\n[ { \"name\": \"core/heading\", \"attributes\": { \"content\": \"This is an Example\" } } ]\n\nWhere each block itself is an object that contains the block name, the block attributes, and the blocks inner blocks."
```